Yoast/yoast-components
# Implement input field component for keyword

## Summary

This PR can be summarized in the following changelog entry:

* Adds a keyword input field component 

## Relevant technical choices:

* Checks the keyword input and displays an error text under the field in case it finds comma separated words
* Saves the result of the check (whether or not to show the error message) in the state

## Test instructions

This PR can be tested by following these steps:

* Run `yarn install`, `yarn start`, go to `http://localhost:3333/`, and go to the `Keyword` tab
* Typing a single word in the input field should not display the error message
* Typing two words separated by whitespace should not display the error message
* Typing two or more words separated by commas should display the error message underneath the input field

Fixes #466
